### ISSUE TITLE
Insatiable doesn't work as advertized

### ISSUE THREAD

**Issue by That-One-Dude** [SlimeKnights/TinkersConstruct#4713]
### Minecraft Version

1.16.5

### Forge Version

36.2.8

### Mantle Version

1.6.152

### Tinkers' Construct Version

3.3.2.324

### Describe your issue

Insatiable gives 1/4 the amount of damage it claims to give.

### Reproduced with only Tinkers' Construct?

No

### Other mods required

Tinkers Archery

### Confirm the following

- [x] Confirm you have checked the roadmap, what's new, and the FAQ for your issue
- [X] Confirm you have searched the tracker to see if your issue already exists
- [X] Confirm you are not using performance enhancers, such as optifine

**Comment by KnightMiner** [comment]
Please clarify this issue. Are you saying Insatiable gives less damage, or insatiable on Tinkers Archery tools gives less damage? I see no reason why tinkers archery would be required for this issue

**Comment by That-One-Dude** [comment]
Insatiable in general deals 1/4 the damage the tooltip and book claims to give. I tested it by giving myself a sword with wooden handles and manyullyn blade, spawned a polar bear, recoreded its health, hit it, recorded the new health, gave myself the insatiable VIII effect, attacked the polar bear again and recorded its health. I found that The difference in damage was 0.5, not the 2 that was claimed by tooltip and book. I found the issue out when looking at the code for the Insatiable Modifier
